common/lib/xmodule/xmodule/course_metadata_utils.py

```diff
@@ -93,7 +93,8 @@ def course_start_date_is_default(start, advertised_start):
     return advertised_start is None and start == DEFAULT_START_DATE
 
 
-def may_certify_for_course(certificates_display_behavior, certificates_show_before_end, has_ended):
+def may_certify_for_course(certificates_display_behavior, certificates_show_before_end,
```

Comment:
nitpick - our convention is to break long lines like so: http://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python_guidelines.html#breaking-long-lines